cmd/new.go

```diff
@@ -44,6 +46,10 @@ func new(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("can not set both node and python flags")
 	}
 
+	if (nodeFlag || pythonFlag) && len(runtimeName) != 0 {
+		return fmt.Errorf("can not set both node/python flags and runtime flag")
```

Comment:
not sure about this I think we should support `deta new --node --runtime nodejs14` ? what do you think?